cfme/intelligence/reports/widgets/__init__.py

```diff
@@ -132,6 +135,46 @@ def create(self, widget_class, *args, **kwargs):
         view.flash.assert_no_error()
         return dashboard_widget
 
+    def import_widget(self, filename, cancel=False):
+        """
+        Import yaml files containing widget data
+        Args:
+            filename (str): Complete path to the file
+            cancel (bool): If true, widgets will not be imported
+        """
+        view = navigate_to(self, "ImportExport")
+        assert view.is_displayed
+
+        view.fill({"upload_file": filename})
+        view.upload_button.click()
+        view.flash.assert_no_error()
+
+        import_view = self.create_view(ImportExportWidgetsCommitView)
```

Comment:
optional:
I think better to add some `wait` as its upload operation may be some time we can face some lag.